spec/unit/plugins/passwd_spec.rb

```diff
@@ -55,7 +56,7 @@
   if "".respond_to?(:force_encoding)
```

Comment:
`.force_encoding` was added in ruby 2.0 (?) so this check is probably safe to remove.